packages/astro/src/runtime/server/render/server-islands.ts

```diff
@@ -67,7 +72,7 @@ let componentExport = ${safeJsonStringify(componentExport)};
 let script = document.querySelector('script[data-island-id="${hostId}"]');
 let data = {
 	componentExport,
-	props: ${safeJsonStringify(props)},
+	props: ${JSON.stringify(propsEncrypted)},
```

Comment:
Any reason to stop using `safeJsonStringify()`?

Reply:
nope, i should be.